src/colors.jl

```diff
@@ -0,0 +1,234 @@
+
+module Colors
+
+using LinearAlgebra
+using ..Tools
+
+export Color3, Color4, Color, colormap
+
+abstract type Color end
+
+struct Color3 <: Color
```

Comment:
a more representative name `RGB_Color` might be helpful. Who knows when you'd want to switch or add `YUV_Color` :)

Reply:
The colors package uses 8 bit color, instead of floats, which is good for images but not for settings where one computes with color.

Reply:
Good idea to change the name, will do so.